passenger_exporter.go

```diff
@@ -22,19 +21,121 @@ import (
 	"github.com/prometheus/common/version"
 )
 
-const (
-	namespace = "passenger_nginx"
+// Info represents the info section of passenger's status.
+type Info struct {
+	PassengerVersion         string       `xml:"passenger_version"`
+	AppCount                 string       `xml:"group_count"`
+	CurrentProcessCount      string       `xml:"process_count"`
+	MaxProcessCount          string       `xml:"max"`
+	CapacityUsed             string       `xml:"capacity_used"`
+	TopLevelRequestQueueSize string       `xml:"get_wait_list_size"`
+	SuperGroups              []SuperGroup `xml:"supergroups>supergroup"`
```

Comment:
is there a reason a lot of these variables are declared as strings as opposed to `ints`?

Reply:
Hmm I haven't actually thought about this. My assumption here is that the original author used strings so that they could parse them and handle any parsing errors on a more fine-grained level. 

I'll look into this though, but at a later time.